src/main/scala/org/scalajs/tools/tsimporter/sc/Printer.scala

```diff
@@ -197,6 +197,10 @@ class Printer(private val output: PrintWriter, outputPackage: String) {
         implicit val withPipe = ListElemSeparator.Pipe
         p"$types"
 
+      case TypeRef.Intersection(types) =>
+        implicit val withPipe = ListElemSeparator.WithKeyword
```

Comment:
This val should called `withWith`. It's awkward, but definitely less than `withPipe` which is a plain lie ;)

Reply:
https://github.com/sjrd/scala-js-ts-importer/pull/46/commits/37921e7bb3bb67b37306dbcd86274bf53d5a7f6a